indico/modules/attachments/controllers/management/base.py

```diff
@@ -152,7 +152,7 @@ def _process(self):
                 if file:
                     self.attachment.file = AttachmentFile(user=session.user, content_type=file.mimetype,
                                                           filename=secure_filename(file.filename, 'attachment'))
-                    self.attachment.file.save(file.file)
+                    self.attachment.file.save(file.stream)
```

Comment:
what is the benefit of using `stream` over `file` property?

Reply:
`file` does not exist, we were adding it in this legacy code that shouldn't even have such side-effects: https://github.com/indico/indico/pull/3088/files#diff-d96ee7e7041ebb8b679fc6200b567dcbL82